src/d3a/d3a_core/util.py

```diff
@@ -455,6 +455,12 @@ def create_subdict_or_update(indict, key, subdict):
     return indict
 
 
+def _create_or_update_subdict(indict, key, subdict):
```

Comment:
```suggestion
def create_or_update_subdict(indict, key, subdict):
```
Minor but its no more a private method and seems to be better qualified for `d3a-interface`

Reply:
Agreed, I will add it do d3a-interface, as I needed the same function in https://github.com/gridsingularity/d3a-web/pull/912 , too, where I will reuse it, too.